nextmv/nextmv/cloud/application.py

```diff
@@ -4,6 +4,7 @@
 import random
 import shutil
 import time
+from collections.abc import Callable
```

Comment:
Hmm, I always use this:

```python
from typing import Callable
```

Which one is preferable? :thinking:

Reply:
```suggestion
from typing import Callable
```

:point_up:  Just in case you want to quickly replace it. But again, ignore if this is the more fitting type. :relaxed:

Reply:
Yeah so according to the [Python docs](https://docs.python.org/3/library/typing.html#annotating-callable-objects):
> Functions – or other [callable](https://docs.python.org/3/glossary.html#term-callable) objects – can be annotated using [collections.abc.Callable](https://docs.python.org/3/library/collections.abc.html#collections.abc.Callable) or deprecated [typing.Callable](https://docs.python.org/3/library/typing.html#typing.Callable). Callable[[int], str] signifies a function that takes a single parameter of type [int](https://docs.python.org/3/library/functions.html#int) and returns a [str](https://docs.python.org/3/library/stdtypes.html#str).

This means that `typing.Callable` is deprecated. Moreover, inspecting the source code, `typing.Callable` has a deprecated annotation and simply points to `collections.abc.Callable`.

Reply:
Interesting, good to know. :+1: